swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = "", condition = 0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
+        if self.condition == 0:
+            return "Probably best for scraps"
+        elif self.condition == 1:
+            return "Pretty heavily used"
+        elif self.condition == 2:
+            return "Moderate wear"
+        elif self.condition == 3:
+            return "Lightly used"
+        elif self.condition == 4:
+            return "Like new condition"
+        elif self.condition == 5:
+            return "Brand new in box/ with tags"
```

Comment:
Conditions can be a float, but this function will only work for whole numbers. If an item has a condition of 3.5, this function will return `None`. How could you adapt these conditionals to work for floats?